content/docs/user-guide/tips-and-tricks.md

```diff
@@ -0,0 +1,17 @@
+# Tips and tricks for DVC Projects
+
+This guide provides general tips and tricks related to DVC, which can be
+utilized while working on a project. Using the practices listed here, you can
+manage your projects with DVC more efficiently.
```

Comment:
Same: no need for this file here any more. It's been extracted to https://github.com/iterative/dvc.org/pull/1748